holobot/sdk/utils/string_utils.py

```diff
@@ -2,19 +2,13 @@ def join(value: tuple[str, ...], separator: str = ""):
     return separator.join(value)
 
 def try_parse_int(value: str | None) -> int | None:
-    if value is None:
-        return None
```

Comment:
Moved to `try` block because exceptions in python are zero cost if no exception is raised.